lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -34,11 +34,22 @@ def process_row(row)
     end
 
     def find_document(row)
-      slug = row.fetch("Slug")
+      slug = row.fetch("Slug")&.strip
+      document_type = row.fetch("Document type")&.strip
 
       return if slug.blank?
 
-      document = Document.find_by(slug: slug)
+      if document_type.blank?
+        documents = Document.where(slug: slug.strip).to_a
+
+        if documents.length > 1
```

Comment:
It feels like this condition is missing a test - can we add one that checks that two document types with the same slug raises the error you expect?

Reply:
Sure, I've included one now.